_includes/use-cases/use-case-transport-create-token-for-file.md

```diff
@@ -0,0 +1,12 @@
+Creates a [token]({% link _docs_integrate/data-model-overview.md %}#token) for a given file that
+corresponds to the given id.
```

Comment:
Parameters, Success and Failure headlines (and descriptions) are missing.

Reply:
This is done in the new branch.